dockers/docker-orchagent/swssconfig.sh

```diff
@@ -8,19 +8,19 @@ function fast_reboot {
       if [[ -f /fdb.json ]];
       then
         swssconfig /fdb.json
-        rm -f /fdb.json
+        mv /fdb.json /fdb.json.1
```

Comment:
mv -f? the .1 file might exist with unexpected permission.

Reply:
Ack